sdks/Cordova.md

```diff
@@ -0,0 +1,136 @@
+# Apache Cordova AeroGear Services
```

Comment:
Should this be changes to AeroGear JS SDK Proposal?

Then call out Web, Cordova and React Native as the targets for this SDK?